src/sentry/api/endpoints/project_ownership.py

```diff
@@ -20,11 +26,29 @@
 HIGHER_MAX_RAW_LENGTH = 200_000
 
 
-class ProjectOwnershipSerializer(serializers.Serializer):
-    raw = serializers.CharField(allow_blank=True)
-    fallthrough = serializers.BooleanField()
-    autoAssignment = serializers.CharField(allow_blank=False)
-    codeownersAutoSync = serializers.BooleanField(default=True)
+class ProjectOwnershipRequestSerializer(serializers.Serializer):
+    raw = serializers.CharField(
+        required=False,
+        allow_blank=True,
+        help_text="Raw input for ownership configuration. Check out [Ownership Rules Documentation](/product/issues/ownership-rules/) to learn more.",
+    )
+    fallthrough = serializers.BooleanField(
+        required=False,
+        help_text="Set to `True` to make everyone an implicit owner if there is no match on explicit rules.",
+    )
+    autoAssignment = serializers.CharField(
+        required=False,
+        allow_blank=False,
+        help_text="""Auto-assignment settings. Here are the available options:
+- Auto Assign to Issue Owner
+- Auto Assign to Suspect Commits
+- Turn off Auto-Assignment""",
```

Comment:
nit: parity with wording.
```suggestion
        help_text="""Auto-assignment settings. The available options are:
- Auto Assign to Issue Owner
- Auto Assign to Suspect Commits
- Turn off Auto-Assignment""",
```
Also, is the body param actually the full sentence like this? b/c if so that feels very strange
```JSON
{"autoAssignment": "Auto Assign to Issue Owner"}
```

Reply:
Yes, the body param is the actual full sentence 😅  I'll add to future API improvements to fix this.